src/main/kotlin/org/sol4k/VersionedTransaction.kt

```diff
@@ -0,0 +1,81 @@
+package org.sol4k
+
+import org.sol4k.Constants.SIGNATURE_LENGTH
+import org.sol4k.exception.SerializationException
+import java.io.ByteArrayOutputStream
+import java.math.BigDecimal
+import java.math.RoundingMode
+import java.util.Base64
+import kotlin.math.max
+
+class VersionedTransaction(
+    val message: Message,
+    val signatures: MutableList<String>,
```

Comment:
Let's have another constructor without this parameter as the user will want to use the `sign` function when creating a transaction from scratch

Reply:
done

Reply:
Not sure what you exactly mean here, could you please refer to a demonstration of `web3.js` or something else here

Reply:
@Tougee when the user creates a Versioned Transaction in web3js, they don't explicitly pass signatures to it. Instead, they call the `sign()` function.
```js
const message = new TransactionMessage({
    payerKey,
    recentBlockhash,
    instructions 
}).compileToV0Message(); 

const transaction = new VersionedTransaction(message); // <--- no signatures are passed here

transaction.sign([payer]); // <--- this call would add the signature
```
Does it make sense? 

I understand, that for the deserialization case, it is convenient to pass signatures. But it is makes the API more complicated for the users who create a Versioned Transaction from scratch.

Reply:
Nice suggestion, this feature will make it more convenient to use.